src/sxastarter/package.json

```diff
@@ -86,6 +86,7 @@
     "scaffold": "ts-node --project tsconfig.scripts.json scripts/scaffold-component.ts",
     "start:connected": "npm-run-all --serial bootstrap --parallel next:dev start:watch-components",
     "start:production": "npm-run-all --serial bootstrap next:build next:start",
+    "run:production": "npm-run-all --serial bootstrap next:start",
```

Comment:
Why do we need to add "run:production" - can't we use "next:start"? This adds bootstrap, but that shouldn't be necessary as it would be run during "build".